### PR TITLE
Add yubikey cli tool

### PR DESCRIPTION
**Pull request by HWataru**
Sometime we need cli tool #1008

### CODE REVIEW

File: yubico-piv-tool.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "homepage": "https://developers.yubico.com/yubico-piv-tool/",
+    "version": "1.5.0",
+    "license": "BSD 2-Clause",
```
Comment (rasa) [inline yubico-piv-tool.json]: Needs a dash

File: yubico-piv-tool.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        "64bit": {
+            "url": "https://developers.yubico.com/yubico-piv-tool/Releases/yubico-piv-tool-1.5.0-win64.zip",
+            "hash": "d51fdc7dc1f717e1b0fafb503e999377b95496c4fa2d1686e37fbbb063f9ca15"
+        },
+        "32bit": {
+            "url": "https://developers.yubico.com/yubico-piv-tool/Releases/yubico-piv-tool-1.5.0-win32.zip",
+            "hash": "74695de467b13d7320b99ac85a04ff24ae18c7d3230f71aa85fda07feba426d2"
+        }
+    },
+    "extract_dir": "",
```
Comment (rasa) [inline yubico-piv-tool.json]: Can be removed

File: yubico-piv-tool.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "bin": [
+        "bin\\yubico-piv-tool.exe"
```
Comment (rasa) [inline yubico-piv-tool.json]: Since they’re both CLI, do these two apps belong in the main bucket instead?

Comment (HWataru) [reply yubico-piv-tool.json]: Oh, CLI tool shoud belong to main bucket ? I didn't know that.

File: yubico-piv-tool.json (hunk 1 above)
Comment (HWataru) [reply yubico-piv-tool.json]: Thank you. But I can't understand what you say.

Comment (rasa) [reply yubico-piv-tool.json]: Sorry, `"BSD 2-Clause"` should be `"BSD-2-Clause"`

File: yubico-piv-tool.json (hunk 3 above)
Comment (rasa) [reply yubico-piv-tool.json]: Yes, can you submit this PR to 
https://github.com/lukesampson/scoop instead?

**Comment by HWataru** [comment]
Thank you for user advice.

I created PR [scoop#2423](https://github.com/lukesampson/scoop/pull/2423)

**Pull request closed without merging**